content/get-started/workshop/04_sharing_app.md

```diff
@@ -52,6 +52,10 @@ In the following image, you can see an example Docker command from Docker Hub. T
 
     To fix this, you need to tag your existing image you've built to give it another name.
 
+   > [!NOTE]
+   >
+   > If you encounter the following error: `denied: requested access to the resource is denied`, you can retry with `$ docker login -u "myusername" -p "mypassword" docker.io`.
```

Comment:
We try to avoid using `-p` because that would include the user's passphrase in shell history; the `docker.io` should not be needed (as it's the default).

But looking at the steps, it looks like step 2 below is to log in; should we make step 2 the first step? I think that's the problem here; we try to push before making sure the user is logged in.